<?xml version="1.0"?>
<launch>
   <rosparam command="load" file="$(find tinymovr_ros)/config/hardware.yaml" />
   <rosparam command="load" file="$(find tinymovr_ros)/config/diff_drive_config.yaml" />

   <node name="tinymovr_joint_iface" pkg="tinymovr_ros" type="tinymovr_joint_iface_node" output="screen"/>
   <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="diff_drive_controller"/>


</launch>
